sympy/ntheory/tests/test_factor_.py

```diff
@@ -619,3 +619,15 @@ def test_is_amicable():
     assert is_amicable(173, 129) is False
     assert is_amicable(220, 284) is True
     assert is_amicable(8756, 8756) is False
+
+
+def test_issue_17676():
+    n = 28300421052393658575  #original issue 17676 problematic input
+    assert sqrt(n)**2 == n
```

Comment:
We should test factorint directly, since that is where the bugfix is.

Reply:
Moved new tests under test_factorint() in the following commit